tests/41end-to-end-keys/08-cross-signing.pl

```diff
@@ -682,7 +682,17 @@
       })->then( sub {
          sync_until_user_in_device_list( $user1, $user2 );
       })->then( sub {
-         matrix_get_e2e_keys( $user1, $user2_id );
+         repeat_until_true {
+            # Wait until user1 sees signatures uploaded by user2.
+            # We repeat_until_true because this is sensitive to replication delays.
+            matrix_get_e2e_keys( $user1, $user2_id )
+            ->then(sub {
```

Comment:
```suggestion
            ->then( sub {
```

looks to be our usual style.